.github/workflows/release.yaml

```diff
@@ -0,0 +1,23 @@
+name: 'Release'
```

Comment:
I'm wondering if we need such action or rather create release in relases and this will cause tag creation

Reply:
You are right - API call will be sufficient.
Thanks!